pkg/command/service/clean.go

```diff
@@ -65,6 +65,11 @@ kperf service clean --namespace-prefix testns / --namespace nsname
 				}
 			}
 
+			if cmd.Flags().Changed("verbose") {
```

Comment:
Suggest to add a variable for `cmd.Flags().Changed("verbose")`, so you do need to check it every time.

Reply:
Also shortcut `-v`